sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -446,11 +446,12 @@ __SYCL_EXPORT pi_result piextMemSampledImageHandleDestroy(
   return pi2ur::piextMemSampledImageHandleDestroy(Context, Device, Handle);
 }
 
-__SYCL_EXPORT pi_result piextMemImageGetInfo(pi_image_mem_handle MemHandle,
+__SYCL_EXPORT pi_result piextMemImageGetInfo(pi_context Context,
+                                             pi_image_mem_handle MemHandle,
```

Comment:
`pi.h` has these two parameters as `const`. Though this was already the case before with `pi_image_mem_handle`, so perhaps the disparity doesn't cause problems.

Reply:
I've removed `const` from both, which matches other handle types